util/src/main/java/tech/pegasys/artemis/util/network/NetworkUtility.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.network;
+
+import com.google.common.base.Suppliers;
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.util.Enumeration;
+import java.util.function.Supplier;
+
+public class NetworkUtility {
+  public static final String INADDR_ANY = "0.0.0.0";
```

Comment:
(optional) Might be nice to replace "0.0.0.0" strings in the codebase with this constant

Reply:
done.